doc/faq.rst

```diff
@@ -313,18 +313,25 @@ being reviewed, you can try:
 
 * follow our :ref:`contribution guidelines <contributing>`, in particular
   :ref:`automated_contributions_policy`, :ref:`filing_bugs`,
-  :ref:`stalled_pull_request` and :ref:`stalled_unclaimed_issues`.
-* complete the provided issue and pull request templates, including a clear and
-  concise description of the issue or motivation for the pull request.
+  :ref:`stalled_pull_request` and :ref:`stalled_unclaimed_issues`,
+* fill in all sections of the issue or pull request template provided by GitHub,
+  including a clear description of the problem or motivation, and any relevant code
+  examples,
 * ensure the title clearly describes the issue or pull request and does not include
   an issue number.
 
 For your pull requests specifically, the following will make it easier to review:
 
-* ensure your PR satisfies all items in the
-  :ref:`Pull request checklist <pr_checklist>`.
-* ensure your PR addresses an issue for which there is clear consensus on the solution.
-* ensure the changes are minimal and directly relevant to the described issue.
+* ensure your PR addresses an issue for which there is clear consensus on the solution
+  (see :ref:`issues_tagged_needs_triage`),
+* ensure the PR satisfies all items in the :ref:`Pull request checklist <pr_checklist>`,
+* ensure the changes are minimal and directly relevant to the described issue,
+* ensure your PR complies with scikit-learn's development conventions (which you can
+  learn about from the docs and by going through similar prior PRs),
+* ensure to only submit code you can explain and maintain, as you might need to defend
```

Comment:
I think this may be included in the AI contributions section and honestly I am not sure writing this several times is not going to stop offending users. These people are unlikely to read or abide by this no matter how many times we write it...